README.md

```diff
@@ -1,17 +1,21 @@
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+    alt="Get it on F-Droid"
+    height="80">](https://f-droid.org/packages/com.phikal.regex/)
```

Comment:
I'm really not a fan of badges in README's -- they make it harder to read, because of the obfuscated HTML.

Reply:
Oh, ok.